AnnoyancesFilter/Cookies/sections/cookies_specific.txt

```diff
@@ -160,6 +160,7 @@ whats-on-netflix.com#$#html.sp-message-open body { overflow-y: visible !importan
 ! !SECTION: Cookies - Popular rules
 !
 !
+moviesroom.pl#%#scriptlet('Cookies.set', 'privacy', '#privacy')
```

Comment:
It does not work on my end.
Besides, the rule has wrong syntax. I will add another one directly.